.github/ISSUE_TEMPLATE/security-assessment.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Security Assessment Request
+about: Request a Security Assessment from SIG Security members for Kubernetes sub-projects
+title: 'REQUEST: Security Assessment for <sub-project-name>'
+labels: area/security-assessment, sig/security
+assignees: 
```

Comment:
Clarifying question: Would this field have info about SIG Security members who are helping with the assessment or would it have a mix of both (requesting K8s sub-project member + SIG security contact) ?

Reply:
Good question. I actually copied this field from some existing templates here: https://github.com/kubernetes/community/tree/master/.github/ISSUE_TEMPLATE where they have kept it empty too :-)

My guess based on this doc: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository is that it is meant as a field to define a default assignee. But can be safely left blank too.